EmployeeManagementService/EmployeeManagementService.API/Controllers/EmployeeController.cs

```diff
@@ -100,5 +100,50 @@ public async Task<IActionResult> UpdateEmployeeInformation([FromBody] EmployeeDT
                 return StatusCode(500, ex.Message);
             }
         }
+
+        [HttpPatch("login")]
+        public async Task<IActionResult> EmployeeLogin([FromBody] EmployeeDTO employee)
+        {
+            try
+            {
+                await _employeeService.EmployeeLogIn(employee.Username, employee.Password);
+
+                return Ok();
+            }
+            catch(Exception ex)
+            {
+                return StatusCode(500, ex.Message);
+            }
+        }
+
+        [HttpPatch("logout/{id}")]
+        public async Task<IActionResult> EmployeeLogput(long id)
+        {
+            try
+            {
+                await _employeeService.EmployeeLogout(id);
+
+                return Ok();
+            }
+            catch (Exception ex)
+            {
+                return StatusCode(500, ex.Message);
+            }
+        }
+
+        [HttpPatch("reset/locked/{id}")]

```

Comment:
lets change route to something like "reset/{id}/locked.

that looks better to me. it means to reset a certain id's locked status.